### ISSUE TITLE
Merge a PR only with explicit implementor consensus (followup)

### ISSUE THREAD

**Issue by saschanaz** [w3c/clipboard-apis#207]
See the discussion in https://github.com/w3c/clipboard-apis/pull/197#issuecomment-1841380176. It has Agenda+ but in a closed PR, so I'm opening a new issue to make things a bit more clear.

**Comment by annevk** [comment]
I strongly agree that we shouldn't make changes to the specification without agreement. Especially here where there was very clearly opposition. That doesn't make any sense.

**Comment by snianu** [comment]
Sorry for the confusion. Our intent was just to document browser behaviors so that all browsers can be spec-compliant, and not force a particular implementation. We discussed this and #206 issue in the EditingWG meeting and [came to a consensus](https://github.com/w3c/clipboard-apis/issues/206#issuecomment-1858499460). Consensus was to incorporate the proposed changes to the spec, but with minor editorial and non-normative updates, which we will be making.

**Comment by snianu** [comment]
Minutes
<smaug> saschanaz: pr landed to spec without consensus
<smaug> saschanaz: we need to ensure pr has consensus from 2 different implementations
6:52 PM edgar: and standards positions aren't resolved
6:53 PM smaug: right now we can fix the issue by doing what we agreed on 206
6:54 PM pr 197 was merged without consensus and that will be fixed in issue 206
6:54 PM smaug: we should avoid landing PRs without consensus
6:55 PM sanketj_: point taken

**Comment by zcorpan** [comment]
The [WHATWG PR template](https://github.com/whatwg/html/blob/main/PULL_REQUEST_TEMPLATE.md) separates "At least two implementers are interested (and none opposed):" from "Implementation bugs are filed:". Maybe something like that could help here?
